### PR TITLE
Update font-koruri to version 20160506

### PR DESCRIPTION
**Pull request by unasuke**
#### Editing an existing cask

- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download font-koruri` is error-free.
- [x] `brew cask style --fix font-koruri` left no offenses.

### CODE REVIEW

File: Casks/font-koruri.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cask 'font-koruri' do
-  version '20151021'
-  sha256 '6627a5a9532efb6c6c26ae5eb9127d2bbb43db0f5decf0b9d2c77a044d4fba85'
+  version '20160506'
+  sha256 '15b72c803af963f72c838053a10bea1ae608b36ac9acaf2b69f7d924ee38181f'
 
-  # github.com/Koruri/Koruri was verified as official when first introduced to the cask
-  url "https://github.com/Koruri/Koruri/releases/download/Koruri-#{version}/Koruri-#{version}.tar.xz"
```
Comment (vitorgalvao) [inline Casks/font-koruri.rb]: Links are still available as github releases. Please keep them that way.

Comment (unasuke) [inline Casks/font-koruri.rb]: Font owner write in release page "Please download from osdn.jp".
We should respect owner's  opinion.

Comment (vitorgalvao) [inline Casks/font-koruri.rb]: Can you point to where they say that?

Comment (unasuke) [inline Casks/font-koruri.rb]: here https://github.com/Koruri/Koruri/releases/tag/Koruri-20160506 (Japanese)
「ダウンロードは こちら からどうぞ」

File: Casks/font-koruri.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+  # osdn.jp/koruri was verified as official when first introduced to the cask
+  url "http://dl.osdn.jp/koruri/65771/Koruri-#{version}.tar.xz"
```
Comment (vitorgalvao) [inline Casks/font-koruri.rb]: `http://dl.osdn.jp/koruri/#{version.after_comma}/Koruri-#{version.before_comma}.tar.xz`

File: Casks/font-koruri.rb [hunk 3]
```diff
@@ -1,11 +1,11 @@
 cask 'font-koruri' do
-  version '20151021'
-  sha256 '6627a5a9532efb6c6c26ae5eb9127d2bbb43db0f5decf0b9d2c77a044d4fba85'
+  version '20160506'
```
Comment (vitorgalvao) [inline Casks/font-koruri.rb]: `20160506,65771`

**Comment by unasuke** [comment]
@vitorgalvao Fixed from your review and passed audit and rubocop. Thank you!

**Pull request merged**